bee-common/bee-packable/src/packable/mod.rs

```diff
@@ -29,27 +29,82 @@ use core::{convert::AsRef, fmt::Debug};
 /// A type that can be packed and unpacked.
 ///
 /// Almost all basic sized types implement this trait. This trait can be derived using the
-/// [`Packable`](bee_packable_derive::Packable) macro. If you need to implement this trait manually, use the provided
-/// implementations as a guide.
+/// [`Packable`](bee_packable_derive::Packable) macro. If you require to implement this trait
```

Comment:
Maybe something like: "The following example shows how to implement this trait manually."